spyder/plugins/editor/confpage.py

```diff
@@ -48,6 +48,8 @@ def setup_page(self):
                                      'indent_guides')
         showcodefolding_box = newcb(_("Show code folding"), 'code_folding')
         linenumbers_box = newcb(_("Show line numbers"), 'line_numbers')
+        breakpoints_box = newcb(_("Show breakpoints"), 'breakpoints_panel',
+                                section='debugger', default=True)
```

Comment:
```suggestion
                                section='debugger')
```

This is incorrect because (almost always) options need to have defaults when they are transient (e.g. when they depend on the user filesystem). Instead, please add this new option to `debugger` section in `spyder/config/main.py`.